src/model/model.ts

```diff
@@ -230,6 +247,23 @@ export class Model {
     return false;
   }
 
+  // removePolicies removes policy rules from the model.
+  public removePolicies(sec: string, ptype: string, rules: string[][]): boolean {
+    const ast = this.model.get(sec)?.get(ptype);
+    for (const rule of rules) {
+      if (this.hasPolicy(sec, ptype, rule)) {
+        if (!ast) {
```

Comment:
You forgot to move it.

Reply:
Sorry, I didn't get you? Where should I move the statement?

Reply:
Move to out of for loop.
```js
const ast = this.model.get(sec)?.get(ptype);
if (!ast) { return false; }
```